config/samples/azure_v1alpha1_postgresqluser.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: azure.microsoft.com/v1alpha1
+kind: PostgreSQLUser
+metadata:
+  name: psqluser-sample2
+spec:
+  server: postgresqlserver-sample
+  dbName: postgresqldatabase-sample
+  resourceGroup: resourcegroup-azure-operators
+  # The Azure Database for PostgreSQL server is created with the 3 default roles defined.
+  # azure_pg_admin
+  # azure_superuser
+  # your server admin user
+  roles:
+    - "azure_pg_admin"
+   # Specify a specific username for the user
+  username: hong
```

Comment:
I think we should comment out the rest of the optional fields (username/adminsecret/adminsecretkeyvault)

Also can you change username to be a generic value instead of Hong? :)

Reply:
This is the standard we follow with the azuresqluser, too! ((Here:))][https://github.com/Azure/azure-service-operator/blob/master/config/samples/azure_v1alpha1_azuresqluser.yaml]

Reply:
@buhongw7583c Can you also comment out the other two optional fields? username and adminSecret? Thanks!

Also thanks for changing the name of postgresqluser :) Good fix! :) 👍